src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue

```diff
@@ -23,9 +23,9 @@
 	<div class="breakout-rooms">
 		<!-- Series of buttons at the top of the tab, these affect all
 		 breakout rooms -->
-		<div v-if="canModerate" class="breakout-rooms__actions">
+		<div v-if="canModerate || isInBreakoutRoom" class="breakout-rooms__actions">
 			<div class="breakout-rooms__actions-group">
-				<NcButton v-if="breakoutRoomsNotStarted"
+				<NcButton v-if="breakoutRoomsNotStarted && canModerate"
```

Comment:
![image](https://user-images.githubusercontent.com/213943/222145832-ab228ddc-0387-4617-ad73-600d3f402d1e.png)


"Start breakout rooms" now shows for moderators even inside the breakout rooms. Should be only on the main room.
Same for stop